src/ai/backend/manager/services/container_registry/actions/delete_container_registry.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass
+from typing import Optional, override
+from uuid import UUID
+
+from ai.backend.manager.actions.action import BaseActionResult
+from ai.backend.manager.data.container_registry.types import ContainerRegistryData
+from ai.backend.manager.services.container_registry.actions.base import ContainerRegistryAction
+
+
+@dataclass
+class DeleteContainerRegistryAction(ContainerRegistryAction):
+    id: UUID
+
+    @override
+    def entity_id(self) -> Optional[str]:
+        return None
```

Comment:
entity_id is missing
```suggestion
        return self.id
```